criu/mount.c

```diff
@@ -1196,7 +1196,8 @@ bool mnt_is_overmounted(struct mount_info *mi)
 
 static int set_is_overmounted(struct mount_info *mi)
 {
-	mnt_is_overmounted(mi);
+	if (mnt_is_overmounted(mi))
```

Comment:
mnt_is_overmounted returns false if a mount is overmounted, why should we return -1 here? Cc: @Snorch

Reply:
True is returned then mount is overmounted  and false if it is not-overmounted. both states are valid, also the state is cached on on mount_info. We call mnt_is_overmounted here to prepare the cache (in next steps of mount restore we slightly corrupt mount tree so to have the right result of mnt_is_overmounted we need to compute it here), there is no sense in checking the return code.

Reply:
Thanks. I will drop this.